### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by hannessolo**

### CODE REVIEW

File: test/test-project/package.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "dependencies": {
+    "babel-plugin-styled-components": "^2.0.7",
```
Comment (icaraps) [inline test/test-project/package.json:3]: Do you have issues if you don't include this dependency ?

File: bin/franklin.js [hunk 2]
```diff
@@ -226,7 +226,8 @@ async function buildCommand({custom}) {
   
   // Use custom index.css if any
   if (custom.css.exists) {
-    fs.copySync(custom.css.path, path.join(clientDir, 'index.css'))
+    const customCss = fs.readFileSync(custom.css.path)
```
[the remark this answers is not in the document]
Comment (hannessolo) [reply bin/franklin.js:229]: Using the CLI to generate the variables seems like a good option. I'll revert this.

File: test/test-project/pages/content/home.page.jsx [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export default function Home() {
```
Comment (icaraps) [inline test/test-project/pages/content/home.page.jsx:1]: Did you verify that

* `.page.client.js` https://vite-plugin-ssr.com/.page.client.js 
*  `page.route.js` https://vite-plugin-ssr.com/.page.route.js 

are working ? Or it's not supported currently ?

Comment (hannessolo) [reply test/test-project/pages/content/home.page.jsx:1]: For custom pages? Only `.page.js` is supported at the moment

**Review by icaraps — APPROVED** [review]

**Comment by icaraps** [comment]
@hannessolo checks have failed, should we still merge ?

**Comment by hannessolo** [comment]
> @hannessolo checks have failed, should we still merge ?

Yep, it seems the failures are because the github app is running out of disk storage (cc @ramboz)

**Pull request merged**